Guideline.md

```diff
@@ -62,7 +62,11 @@ Knowing your audience is critical to ensuring that meeting room layouts, menus,
 What you write and what you say can enhance the dignity of people with disabilities and promote positive attitudes about their abilities. [Examples of Affirmative Phrases](http://www.adahospitality.org/sites/default/files/sites/default/files/Choosing%20Words%20with%20Dignity.pdf) (Do use) and Negative Phrases (Don't use).
 
 #### Avoid gendered pronouns
-Avoid using male pronouns ("he", "him", "his") in circumstances that describe people of any gender. Find [examples of nonsexist language](http://geekfeminism.wikia.com/wiki/Nonsexist_language). 
+Avoid using male pronouns ("he", "him", "his") in circumstances that describe people of any gender. Find [examples of nonsexist language](http://geekfeminism.wikia.com/wiki/Nonsexist_language).
```

Comment:
This reads like 'avoid using male pronouns [...] for anyone' which conflicts with the latter advice about using preferred pronouns

I think by 'people of any gender' you mean 'people who could be any gender'?

Reply:
I'm under the assumption that the paragraph at line 65 is talking about (two or more) *people*, while I'm adding paragraphs at line 67-69 to talk about *individuals*.

I didn't want to change the original paragraph (the diff is only whitespace) unless I had some feedback and knew how to better clarify meaning. Once I have that I can attempt to improve grammar flow between the paragraphs.

Reply:
```diff
-Avoid using male pronouns ("he", "him", "his") in circumstances that describe people of any gender.
+Avoid using male pronouns ("he", "him", "his") as a collective term in circumstances that describe people of any, or multiple, genders.
```

?